pkg/tbtc/node.go

```diff
@@ -345,3 +347,158 @@ func (n *node) joinDKGIfEligible(seed *big.Int, startBlockNumber uint64) {
 		logger.Infof("not eligible for DKG with seed [0x%x]", seed)
 	}
 }
+
+// joinSigningIfEligible takes a message and undergoes the process of tECDSA
+// signing if this node's operator proves to control some signers of the
+// requested wallet. This is an interactive process, and joinSigningIfEligible
+// can block for an extended period of time while it completes the operation.
+//
+// TODO: Ultimately, one client will handle multiple signature requests for
+//       multiple wallets. Because of that, logging within that function
+//       must use the context of the given signature request.
+func (n *node) joinSigningIfEligible(
+	message *big.Int,
+	walletPublicKey *ecdsa.PublicKey,
+	startBlockNumber uint64,
+) {
+	logger.Infof(
+		"checking eligibility for signature of message [%v]",
+		message,
+	)
+
+	if signers := n.walletRegistry.getSigners(
+		walletPublicKey,
+	); len(signers) > 0 {
+		logger.Infof(
+			"joining signature of message [%v] and "+
```

Comment:
```suggestion
			"joining signature of message [%v] "+
```

Reply:
Addressed that in https://github.com/keep-network/keep-core/pull/3228/commits/10e6f8ca69e15022c1501ca04d5253b037645d06